frontend/src/app/offers/offers.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { Headers, Http, Response, RequestOptions } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class OffersService {
+  private url = 'http://localhost:8000/api/offers/';
+
+  constructor (
+    private http: Http
+  ) {}
+
+  getOffers() {
+    let headers: Headers = new Headers({
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*'
+    });
+    let options: RequestOptions = new RequestOptions({ headers: headers });
```

Comment:
You need to send ` { withCredentials: true }` here (see for reference: `auth.service.ts`